packages/grid/rollup.config.js

```diff
@@ -1,57 +1,50 @@
-import resolve from '@rollup/plugin-node-resolve';
 import typescript from 'rollup-plugin-typescript2';
 import cleaner from 'rollup-plugin-cleaner';
 import sourceMaps from 'rollup-plugin-sourcemaps';
 import { terser } from 'rollup-plugin-terser';
 import dts from 'rollup-plugin-dts';
 import command from 'rollup-plugin-command';
-import pkg from './package.json';
+import pkg from './data-grid/package.json';
 
 // dev build if watching, prod build if not
 const production = !process.env.ROLLUP_WATCH;
 export default [
   {
-    input: 'src/index.ts',
+    input: './data-grid/src/index.ts',
     output: [
       {
-        file: 'dist/index-esm.js',
+        file: './data-grid/dist/index-esm.js',
         format: 'esm',
         sourcemap: !production,
       },
       {
-        file: 'dist/index-cjs.js',
+        file: './data-grid/dist/index-cjs.js',
         format: 'cjs',
         sourcemap: !production,
       },
     ],
 
     external: [...Object.keys(pkg.peerDependencies || {})],
     plugins: [
-      resolve({
-        resolveOnly: [/^@material-ui\/x-.*$/], // we bundle x-license and x-grid-modules
-      }),
       production &&
         cleaner({
-          targets: ['./dist/'],
+          targets: ['./data-grid/dist/'],
         }),
-      typescript({ tsconfig: 'tsconfig.build.json' }),
+      typescript({ tsconfig: 'tsconfig.json' }),
       !production && sourceMaps(),
       production && terser(),
     ],
   },
   {
-    input: './dist/index.d.ts',
-    output: [{ file: 'dist/data-grid.d.ts', format: 'es' }],
+    input: './data-grid/dist/data-grid/src/index.d.ts',
+    output: [{ file: './data-grid/dist/data-grid.d.ts', format: 'es' }],
     plugins: [
       dts(),
-      command(
-        `cat ../x-grid-modules/dist/x-grid-modules.d.ts ./dist/data-grid.d.ts > ./dist/data-grid.all.d.ts`,
-        {
-          exitOnFail: true,
-          wait: true,
-        },
-      ),
       !production && sourceMaps(),
+      production && command([`rm -rf ./data-grid/dist/data-grid/`, `rm -rf ./data-grid/dist/_modules_ `, `rm -rf ./data-grid/dist/x-grid`], {
+        exitOnFail: true,
+        wait: true,
+      }),
     ],
   },
 ];
```

Comment:
I personally think that we shouldn't have a `/grid` folder, that all our packages should be flattened under  `/packages/`. It breaks the sorting but makes it easier to discover all the packages and to navigate in the filesystem with the command line. I put more value in the latter. But not everybody will agree.

Under this above assumption, I think that it would be easier to understand each package by keeping the rollup file inside package folder. We can use the `cp` approach of https://github.com/mui-org/material-ui-x/pull/343/files#diff-04df1612bca87eae32a2bc4d2c5562e6R42 to do that.

Reply:
The grid is one big package source that publish 2 sub packages
The rollup files don't work if placed in the sub folders

Reply:
I'd like to take out the data-gen pkg as it's not part of the component

Reply:
> I'd like to take out the data-gen pkg as it's not part of the component

If we do that, I think that I can get behind the change. It still find it confusing to have /grid/x-grid and grid/data-grid, but at least, it's one more package under /packages/ :).